src/ausweiskopie/resources/translations.csv

```diff
@@ -40,4 +40,7 @@ NPA_BACK;1. New German Identity Card;1. Neuer Personalausweis
 VORLAEUFIG;4. Temporary German Identity Card;4. Vorläufiger Personalausweis
 GERMAN_PASSPORT;5. German biometric Passport (since March 2017);5. Reisepass (ab März 2017)
 NO_BACK;3. (No back);3. (Keine Rückseite)
-VORLAEUFIG_BACK;2. Temporary German Identity Card;2. Vorläufiger Personalausweis
\ No newline at end of file
+VORLAEUFIG_BACK;2. Temporary German Identity Card;2. Vorläufiger Personalausweis
+CH_NIDK_2023;6. Swiss National Identity Card (since 2023);6. Schweizerische Identitätskarte (ab 2023)
+CH_NIDK_2023_BACK;4. Swiss National Identity Card (since 2023);4. Schweizerische Identitätskarte (ab 2023)
+CH_PLACE_OF_ORIGIN;Place of Origin (CH);Heimatort / Bürgerort (CH)
```

Comment:
I'd prefer if CH is spelled out to Swiss

Reply:
Sure, can change it

Reply:
Are you find with CH in the identifiers?

Reply:
Yes

Reply:
This is resolved